plugins/org.jkiss.dbeaver.data.transfer.ui/OSGI-INF/l10n/bundle_de.properties

```diff
@@ -1,4 +1,9 @@
+extension-point.org.jkiss.dbeaver.dataTransferConfigurator.name = Daten\u00FCbertragung Konfigurationen
```

Comment:
Why is it not "Konfiguratoren"?

Reply:
Where the string is showed in dbeaver? "Konfiguratoren" is the configurator ui and "Konfigurationen" is the configuration data itself at whole

Reply:
No, I can't find it in UI. But it still - data Transfer Configurator not configuration.

Reply:
I changed it to "Konfigurator". But why is there a string that is not displayed in the user interface?

Reply:
For historical reasons.
And again: Why is it not "Konfiguratoren"? Why is it not a plural form?

Original string is:
`extension-point.org.jkiss.dbeaver.dataTransferConfigurator.name = Data Transfer UI Configurators`

Reply:
I changed it to Konfiguratoren